pkg/proxy/iptables/proxier.go

```diff
@@ -1244,6 +1234,17 @@ func (proxier *Proxier) syncProxyRules() {
 			writeLine(proxier.natRules, args...)
 		}
 
+		// Next, redirect all src-type=LOCAL -> LB IP to the service chain for externalTrafficPolicy=Local
+		// This prevents traffic originating from the host, but not pods, to be redirected to the service correctly,
+		// otherwise traffic to LB IPs are dropped if there are no local endpoints.
+		args = append(args[:0], "-A", string(svcXlbChain))
+		writeLine(proxier.natRules, append(args,
+			"-m", "comment", "--comment", fmt.Sprintf(`"masquerade LOCAL traffic for %s LB IP"`, svcNameString),
```

Comment:
Mentioned here https://github.com/kubernetes/kubernetes/issues/65387#issuecomment-490637949 that this masquerade rule is not needed since we masquerade at the `KEP-SEP*` chains. Thoughts on removing in a follow-up PR?

Reply:
ACK if we can prove it :)

Reply:
I actually like the explicitness here, will leave it for now